### ISSUE TITLE
[EN][Professions] Nodes not respawning

### ISSUE THREAD

**Issue by sebastianluczak** [Atlantiss/BugTracker#6425]
As of today new bug appeared. Professions affected, tested myself:
- Herbalism
- Mining

Professions probably affected, reported by others:
- Skinning
- Fishing

**Description of bug:**
After collecting herb or/and mining a node it fails to disappear, staying on the ground as clickable object. After second attemp of harvesting nothing happens. No new nodes are spawning and current node is not despawned.

**How it should work:**
After collecting a node it should disappear and another similar should appear on random spot on current location.

What's worth noticing - people also report strange bug in Instances, being able to multi-loot some or all of the items dropped from bosses. Can or cannot be related.

**Comment by redneckdad** [comment]
Yeah seen this first by my wife was herbin, so i logged in and mining was messed up too. it was weird that i got mining lvl credit, but no ore was dropping. I NEED MY ORE! lol Plz try to get fixed, Just started this PS 2 or 3 days ago, came from warmane.(( :(  ))) ( IT REALLY SUCKS THERE )) its where all bad ppl from wow spend their time in hell,lol  TY plz fix

**Comment by sebastianluczak** [comment]
After the night situation seems to get back to normal.

Haven't encountered empty node yet, playing for 2 hours.
